ximena_subieta/space_challenge/src/U1.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * This is U1 class that extends of Rocket
+ */
+public class U1 extends Rocket {
+
+    public U1 (){
+        cost = 100;
```

Comment:
please first define your properties 

private in cost;

Reply:
I am extending no need to declare attributes, am I wrong?